README.md

```diff
@@ -1,75 +1,97 @@
 # 👋 Welcome to skore
 
 ![ci](https://github.com/probabl-ai/skore/actions/workflows/ci.yml/badge.svg?event=push)
-![python](https://img.shields.io/badge/python-3.11%20|%203.12-blue?style=flat&logo=python)
+![python](https://img.shields.io/badge/python-3.9%20%7C%203.10%20%7C%203.11%20%7C%203.12-blue?style=flat&logo=python)
+![pypi](https://img.shields.io/pypi/v/skore)
+![license](https://img.shields.io/pypi/l/skore)
 
-`skore` allows data scientists to create tracking and visualization from their Python code:
-1. Users can store objects of different types: python lists and dictionaries, `numpy` arrays, `scikit-learn` fitted models, `matplotlib`, `altair`, and `plotly` figures, etc. Storing some values over time allows one to perform **tracking** and also to **visualize** them:
-2. They can visualize these stored objects on a dashboard. The dashboard is user-friendly: objects can easily be organized.
-3. This dashboard can be exported into a HTML file.
+With `skore`, data scientists can:
+1. Store objects of different types from their Python code: python lists, `scikit-learn` fitted pipelines, `plotly` figures, and more.
+2. **Track** and  **visualize** these stored objects on a user-friendly dashboard.
+3. Export the dashboard to a HTML file.
 
-These are only the first features of `skore`'s roadmap.
-`skore` is a work in progress and, on the long run, it aims to be an all-inclusive library for data scientists.
+These are only the first features: `skore` is a work in progress and aims to be an end-to-end library for data scientists.
 Stay tuned!
 
-<p align="center">
-    <img width="100%" src="https://github.com/sylvaincom/sylvaincom.github.io/blob/master/files/probabl/skore/2024_10_08_skore_demo.gif"/>
-</p>
+![GIF: short demo of `skore`](https://raw.githubusercontent.com/sylvaincom/sylvaincom.github.io/master/files/probabl/skore/2024_10_14_skore_demo.gif)
 
 ## ⚙️ Installation
 
-You can install `skore` by using `pip`:
+First of all, we recommend using a [virtual environment (venv)](https://docs.python.org/3/tutorial/venv.html). You need `python>=3.9`.
+
+Then, you can install `skore` by using `pip`:
 ```bash
 pip install -U skore
 ```
 
+You can check `skore`'s latest version on
```

Comment:
why would the users do that? pip should already install last version.  
As we aim at having a shortest readme as possible, I'm not sure that it is necessary, all the more that there is already a badge at the top leading to pypi.

Reply:
ok, removing